config/boot.rb

```diff
@@ -1,26 +1,14 @@
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
+
+require 'bundler/setup' # Set up gems listed in the Gemfile.
+
 require 'rubygems'
 require 'bundler/setup'
```

Comment:
looks like we could get rid of one of these `require 'bundler/setup'` lines?

Reply:
good call!!